packages/core/src/components/pane-layout/pane-layout.tsx

```diff
@@ -0,0 +1,434 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Siemens AG
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {
+  Component,
+  Element,
+  forceUpdate,
+  h,
+  Host,
+  Listen,
+  Prop,
+  State,
+  Watch,
+} from '@stencil/core';
+import { Composition } from '../pane/pane';
+import { applicationLayoutService } from '../utils/application-layout';
+import { matchBreakpoint } from '../utils/breakpoints';
+
+/**
+ * @since 2.1.0
+ */
+@Component({
+  tag: 'ix-pane-layout',
+  styleUrl: 'pane-layout.scss',
+  shadow: true,
+})
+export class Panes {
+  @Element() hostElement: HTMLIxPaneLayoutElement;
+
+  /**
+   * Choose the layout of the panes
+   */
+  @Prop() layout: 'full-height-left-right' | 'full-width-top-bottom' =
```

Comment:
Maybe we can find simpler strings for this prop like e.g. 'left-to-right' or 'vertical'?